### ISSUE TITLE
Caret doesn't open via associated .md files.

### ISSUE THREAD

**Issue by AzavarSteve** [careteditor/caret#192]
When double-clicking any .md file it opens the Windows "open with" dialog to open Caret, but clicking okay, returns you to the "open with" dialog. (endlessly)  I can open the file using the Caret editor File Open dialog fine.

I'm not sure if this is an isolated issue to my machine, but I've uninstalled and reinstalled to no avail.

Caret v1.11.2
OS: Windows 10

**Comment by astoilkov** [comment]
@AzavarSteve Thanks for reporting. We will investigate.

@borekb Have you experienced similar issue on your Windows machine?

**Comment by borekb** [comment]
Yes. I don't have a clean install of Caret now but I remember setting up Caret as a default program for .md files was a struggle.
